packages/taco/test/conditions/base/json.test.ts

```diff
@@ -0,0 +1,104 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import { describe, expect, it } from 'vitest';
+
+import {
+  JsonCondition,
+  JsonConditionProps,
+  jsonConditionSchema,
+  JsonConditionType,
+} from '../../../src/conditions/base/json';
+
+describe('JsonCondition', () => {
+  describe('validation', () => {
+    it.each([
+      [
+        'object',
+        { store: { book: [{ price: 10.5 }] } },
+        '$.store.book[0].price',
+        10.5,
+      ],
+      ['array', [1, 2, 3, 4, 5], '$[2]', 3],
+      ['number', 42, undefined, 42],
+      ['string', 'hello world', undefined, 'hello world'],
+      ['boolean', true, undefined, true],
```

Comment:
How about adding a JSON-looking string (not a dict) ? e.g. `"{ store: { book: [{ price: 10.5 }] } }"`
Also, it seems the first element of all these test cases (`type`?) is not used.